packages/skills/ml_train/README.md

```diff
@@ -0,0 +1,3 @@
+# ML Skill
```

Comment:
Remove please. We are no longer doing Readmes other than main readme. All descriptions should be in the docs. Then we have one place where we update info and not several :)